### ISSUE TITLE
Inaccurate NostalgicTweaks configuration

### ISSUE THREAD

**Issue by JuniperIsSleepy** [Vladivaston/Minecraft-Console-Edition#3]
Upon comparing directly with tu30 (what I presume this is based on), I find that many of the chosen visual and gameplay changes were innacurate to the console editions, and the list is a varied and sporadic set of changes, so I am providing you with the edited config file I conjured instead. Feel free to review it.
[nostalgic_tweaks.json](https://github.com/user-attachments/files/16401264/nostalgic_tweaks.json)
I am also providing a tweaked options config for accuracy, especially regarding fog.
[options.txt](https://github.com/user-attachments/files/16401267/options.txt)
If legacy4js fog settings do not transfer with this, set the fog start to 2 chunks and the fog end distance to 190% and have Toggle Sneak on by default.

**Comment by omo50** [comment]
ill look into this tomorrow, thanks

**Comment by JuniperIsSleepy** [comment]
Yeah. By the way, i would look into the item obliterator mod if you'd like to make items from tu31 onwards not be obtainable or appear in any menus besides commands.

Also, I have heard that polytone can be used to make water colors universal. Might be worth looking into.
